### PR TITLE
Document that size_bytes is optional for clients but not for lakeFS

### PR DESCRIPTION
**Pull request by arielshaqed**

### CODE REVIEW

File: api/swagger.yml [hunk 1]
```diff
@@ -320,6 +320,10 @@ components:
         size_bytes:
           type: integer
           format: int64
+          description: |
+            The number of bytes in the object.  When lakeFS always populates
+            this field when returning ObjectStats.  This field is optional
```
Comment (rahij) [inline api/swagger.yml:325]: Should the second sentence just be "LakeFS always populates this field when returning ObjectStats"?

Comment (arielshaqed) [reply api/swagger.yml]: Thanks!  Sometimes words are difficult :-/

**Review by arielshaqed — COMMENTED** [review]
Thanks!

**Review by itaiad200 — APPROVED** [review]

**Comment by arielshaqed** [comment]
Thanks!

**Pull request merged**